config/config/safety_center.yml

```diff
@@ -0,0 +1,150 @@
+    color: yellow
+    icon: fas fa-user-clock
+  reviewed:
+    name: Reviewed
+    description: This report has been investigated and appropriate action taken.
+    color: green
+    icon: fas fa-check-circle
+  responded:
+    name: Reporter Responded
+    description: This report has had a response from the reporter.
+    color: yellow
+    icon: fas fa-user-edit
+  closed:
+    name: Closed
+    description: This report has been closed. No further action may be taken.
+    color: tertiary
+    icon: fas fa-times-circle
+
+outcomes:
+  upheld:
+    name: Upheld
+    description: The report is upheld; the reporter's classification of the content is correct; in the case of an
+      appeal, the appeal is upheld and the action taken has been reversed.
+    user_facing:
+      illegal: &illegal_upheld
+        content: our community team agreed with your report and how you classified this content, and have taken
+          appropriate action.
+      abusive: *illegal_upheld
+      copyright: *illegal_upheld
+      appeal:
+        content: our community team agreed with your appeal and have reversed the action taken in your case.
+  actionable:
+    name: Actionable
+    description: The content is actionable but the reported classification is not correct. NOT applicable to appeals.
+    user_facing:
+      illegal: &illegal_actionable
+        content: our community team agreed that the content you reported was actionable and have taken appropriate
+          action, but have changed your classification of the content for reporting purposes.
+      abusive: *illegal_actionable
+      copyright: *illegal_actionable
+      appeal: ~
+  disputed:
+    name: Disputed
+    description: The reporter's classification is disputed; the content does not appear to be actionable; in the case of
+      an appeal, the action taken was appropriate.
+    user_facing:
+      illegal: &illegal_disputed
+        content: our community team have reviewed your report and have decided that the content you reported is not
+          actionable. This does not mean that the content cannot be removed by community moderators for breaking
+          community-specific rules, just that it does not breach our Content Policy.
+      abusive: *illegal_disputed
+      copyright: *illegal_disputed
+      appeal:
+        content: our community team have reviewed your appeal and have decided that the action taken in your case was
+          appropriate.
+
+report_types:
+  illegal:
+    enabled: true
+    name: Report of illegal content
+    description: illegal harmful content
+  abusive:
+    enabled: true
+    name: Report of abusive content
+    description: abusive or otherwise harmful content
+  copyright:
+    enabled: true
+    name: Report of copyright infringement
+    description: copyright infringement
+  appeal:
+    enabled: true
+    name: Classification Appeal
+    description: an appeal regarding how we've handled your content
+

```

Comment:
Suggestion: comment citing Ofcom for this list?  (Signpost for other deployments who need to comply with OSA, and indicator that we didn't just make this up.)